src/Symfony/Bridge/Doctrine/SchemaListener/LockStoreSchemaListener.php

```diff
@@ -30,20 +29,12 @@ public function postGenerateSchema(GenerateSchemaEventArgs $event): void
     {
         $connection = $event->getEntityManager()->getConnection();
 
-        $storesIterator = new \ArrayIterator($this->stores);
-        while ($storesIterator->valid()) {
-            try {
-                $store = $storesIterator->current();
-                if (!$store instanceof DoctrineDbalStore) {
-                    continue;
-                }
-
-                $store->configureSchema($event->getSchema(), $this->getIsSameDatabaseChecker($connection));
-            } catch (InvalidArgumentException) {
```

Comment:
we still need catch the exception to not reintroduce the bug that #50761 tried to fix